portal-web/test/functional/com/liferay/portalweb/paths/portlet/publications/PublicationsChanges.path

```diff
@@ -156,6 +156,34 @@
 	</td>
 <td></td>
 </tr>
+
+<!--SHARE ACCESS-->
+
+<tr>
+	<td>COPY_URL</td>
+	<td>//div[contains (@class,'input-group-item-shrink')]//button[contains(text(),'Copy')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ENABLE_SHARE_LINK_TOGGLE_SWITCH</td>
+	<td>//span[contains(.,'${key_switchText}')]/..//span[contains(@class, 'toggle-switch-check-bar')]//input[contains(@type, 'checkbox')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>SHARE_ACCESS_BUTTON</td>
+	<td>//div/button[contains(@class, 'sticker-user-icon') and (@title = 'Publication Sharing Disabled')]</td>
```

Comment:
Can you remove the title attribute to make it more usable? Since it will change after we enable publication sharing.